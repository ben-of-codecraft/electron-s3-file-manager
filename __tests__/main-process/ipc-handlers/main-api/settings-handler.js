const {mockIpcMainApi} = require('../../utils');
const {
	api,
} = require('../../../../src/main-process/preload/utils');

beforeAll(() => {
	mockIpcMainApi();
});

describe('ipc main api settings handler', () => {
	test('get initial s3 settings', async () => {
		const settings = await api.getSettings();

		expect(settings).toMatchSnapshot();
	});
});
