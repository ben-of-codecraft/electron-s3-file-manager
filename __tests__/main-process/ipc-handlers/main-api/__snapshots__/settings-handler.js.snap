// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ipc main api settings handler get initial s3 settings 1`] = `null`;
